### PR TITLE
[HotFix] create claim action

### PR DESCRIPTION
**Pull request by kubiak85**

### CODE REVIEW

File: extensions/canvasMedical/v1/actions/createClaim/config/fields.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const fields = {
     label: 'Created',
     description:
       'The field indicating when this claim resource was created, accepting a date in the YYYY-MM-DD format.',
-    type: FieldType.DATE,
+    type: FieldType.STRING,
```
Comment (rahulkeerthi) [inline extensions/canvasMedical/v1/actions/createClaim/config/fields.ts:39]: Why this change?

Comment (kubiak85) [reply extensions/canvasMedical/v1/actions/createClaim/config/fields.ts:39]: I wanted to keep the standard with docs

Comment (kubiak85) [reply extensions/canvasMedical/v1/actions/createClaim/config/fields.ts]: Ok
